R/export.R

```diff
@@ -84,6 +87,15 @@ export <- function(
     assets_version <- assets_version()
   }
 
+  wasm_packages_val <- wasm_packages %||% sys_env_wasm_packages()
+  wasm_packages <- as.logical(wasm_packages_val)
+  if (is.na(wasm_packages)) {
+    cli::cli_abort(c(
+      "x" = "Could not parse `wasm_packages` value: {.code {wasm_packages_val}}"
+    ))
```

Comment:
For `cli_abort()` we generally use billets only for additional information or messages. 

```suggestion
    cli::cli_abort(
      "Could not parse `wasm_packages` value: {.code {wasm_packages_val}}"
    )
```